src/resources/base.js

```diff
@@ -0,0 +1,4 @@
+/** @provideGoog */
```

Comment:
This doesn't seem to work.

When debugging `compilerInput.getProvides()` returns 0 provides for this file. If I understand @provideGoog properly, it should be providing `goog.provide` as there is not other method to provide that during bootstrapping.

@matrixfrog, anything obviously wrong with this approach. I am trying to replicate `base.js` so that the Closure compiler is aware of `goog.require` as a provided function symbol.